dozer-types/src/errors/types.rs

```diff
@@ -59,4 +59,12 @@ pub enum DeserializationError {
     Utf8(#[from] std::str::Utf8Error),
     #[error("Failed to convert type due to json numbers being out of the f64 range")]
     F64TypeConversionError,
+    #[error("Unknown SSL mode: {0}")]
+    UnknownSslMode(String),
+    #[error("Invalid connection url for Postgres configuration: {0}")]
+    InvalidConnectionUrl(String),
```

Comment:
It's better if we can keep the original error enum/struct instead of converting it to string, so we don't lose information.

Reply:
Sure